solidity/contracts/bridge/BridgeGovernance.sol

```diff
@@ -0,0 +1,1523 @@
+            movingFundsTxMaxTotalFee,
+            movingFundsDustThreshold,
+            movingFundsTimeoutResetDelay,
+            movingFundsTimeout,
+            movingFundsTimeoutSlashingAmount,
+            movingFundsTimeoutNotifierRewardMultiplier,
+            movedFundsSweepTxMaxTotalFee,
+            movingFundsData.getNewMovedFundsSweepTimeout(),
+            movedFundsSweepTimeoutSlashingAmount,
+            movedFundsSweepTimeoutNotifierRewardMultiplier
+        );
+        movingFundsData.finalizeMovedFundsSweepTimeoutUpdate(
+            governanceDelays[0]
+        );
+    }
+
+    /// @notice Begins the moved funds sweep timeout slashing amount update process.
+    /// @dev Can be called only by the contract owner.
+    /// @param _newMovedFundsSweepTimeoutSlashingAmount New moved funds sweep timeout slashing amount.
+    function beginMovedFundsSweepTimeoutSlashingAmountUpdate(
+        uint96 _newMovedFundsSweepTimeoutSlashingAmount
+    ) external onlyOwner {
+        movingFundsData.beginMovedFundsSweepTimeoutSlashingAmountUpdate(
+            _newMovedFundsSweepTimeoutSlashingAmount
+        );
+    }
+
+    /// @notice Finalizes the moved funds sweep timeout slashing amount update process.
+    /// @dev Can be called only by the contract owner, after the governance
+    ///      delay elapses.
+    function finalizeMovedFundsSweepTimeoutSlashingAmountUpdate()
+        external
+        onlyOwner
+    {
+        (
+            uint64 movingFundsTxMaxTotalFee,
+            uint64 movingFundsDustThreshold,
+            uint32 movingFundsTimeoutResetDelay,
+            uint32 movingFundsTimeout,
+            uint96 movingFundsTimeoutSlashingAmount,
+            uint256 movingFundsTimeoutNotifierRewardMultiplier,
+            uint64 movedFundsSweepTxMaxTotalFee,
+            uint32 movedFundsSweepTimeout,
+            ,
+            uint256 movedFundsSweepTimeoutNotifierRewardMultiplier
+        ) = bridge.movingFundsParameters();
+        // slither-disable-next-line reentrancy-no-eth
+        bridge.updateMovingFundsParameters(
+            movingFundsTxMaxTotalFee,
+            movingFundsDustThreshold,
+            movingFundsTimeoutResetDelay,
+            movingFundsTimeout,
+            movingFundsTimeoutSlashingAmount,
+            movingFundsTimeoutNotifierRewardMultiplier,
+            movedFundsSweepTxMaxTotalFee,
+            movedFundsSweepTimeout,
+            movingFundsData.getNewMovedFundsSweepTimeoutSlashingAmount(),
+            movedFundsSweepTimeoutNotifierRewardMultiplier
+        );
+        movingFundsData.finalizeMovedFundsSweepTimeoutSlashingAmountUpdate(
+            governanceDelays[0]
+        );
+    }
+
+    /// @notice Begins the moved funds sweep timeout notifier reward multiplier update process.
+    /// @dev Can be called only by the contract owner.
+    /// @param _newMovedFundsSweepTimeoutNotifierRewardMultiplier New moved funds sweep timeout notifier reward multiplier.
```

Comment:
We could break the description into more lines here as well.

Reply:
[caa12fa](https://github.com/keep-network/tbtc-v2/pull/329/commits/caa12fa98518fff3ea0aad3a420e624f65647e00)